test/e2e/framework/util.go

```diff
@@ -4432,6 +4432,16 @@ func getKubeletPods(c clientset.Interface, node, resource string) (*v1.PodList,
 	return result, nil
 }
 
+// IPProtocol is the type to hold IP protocol.
+type IPProtocol string
```

Comment:
We don´t need this if we merge https://github.com/kubernetes/kubernetes/pull/80398/

Reply:
That'd be great

Reply:
it was merged , you can use `framework.TestContext.IPFamily`

Reply:
Is it OK to assume ```framework.TestContext.IPFamily``` is always set to either ```ipv4``` or ```ipv6```?

Reply:
is it right now, but it can change if we want to consider `dual-stack` as an IPFamily, I would be conservative and don assume it